template/docu/docu_05variadic_02.cc

```diff
@@ -74,23 +75,95 @@ TEST(Variadic, Filter) {
     static std::string s() { return "(" + A::s() + "+" + B::s() + ")"; }
   };
       
-TEST(Variadic, Reduce) {
+TEST(TemplateMetaprogramming, Reduce) {
   struct A { static std::string s() { return "A"; } };
   struct B { static std::string s() { return "B"; } };
   struct C { static std::string s() { return "C"; } };
   EXPECT_EQ("(A+(B+C))",
-            (bricks::variadic::reduce<concatenate_s, std::tuple<A, B, C>>::s()));
+            (bricks::metaprogramming::reduce<concatenate_s, std::tuple<A, B, C>>::s()));
 }
     
   // Combine.
```

Comment:
This one is for `make docu`: Without two leading spaces this comment line won't make it to `README.md`.

Whereas for example, the test name doesn't have to be there, and thus is not indented. Agree it looks a bit odd, but does the job well so far.